client/src/components/CompaniesTable/index.tsx

```diff
@@ -17,23 +13,19 @@ const columns: GridColDef[] = [
   { field: "krs", headerName: "KRS", width: 250 },
 ];
 
-const rows: Company[] = [
-  {
-    id: 1,
-    name: "Revolve Healthcare",
-    address: "Katowice, ul. Porcelanowa 23 40-246",
-    krs: "0000972657",
-  },
-  {
-    id: 2,
-    name: "Neubloc Polska",
-    address: "Katowice, ul. Grabowa 2 40-172",
-    krs: "0000335382",
-  },
-];
-
 function CompaniesTable() {
   const [selectedRowId, setSelectedRowId] = useState<number | null>(null);
+  const [rows, setRows] = useState<Company[]>([]);
+  const axiosInstance: Axios = axios.create({
+    baseURL: "http://localhost:3000",
+  });
+
+  useEffect(() => {
+    axiosInstance.get("/api/companies").then((response) => {
```

Comment:
nie używaj axiosa bezpośrednio w komponentach, zrób to w thunkach - od tego jest `createAsyncThunk`.
Wtedy też `rows` nie będzie trzymane w stanie komponentu, tylko w Reduxie